src/main/java/tincaalina/BrowserManager.java

```diff
@@ -1,14 +1,13 @@
-package AlinaTinca;
+package tincaalina;
```

Comment:
You already have an alinatinca package with this class in main branch. Please update your branch with last changes and remove this duplicated package. Thanks!

Reply:
done